python/cpstl/datatype/heap/heap.py

```diff
@@ -83,6 +83,7 @@ def heapify(self, node, size_offset=0):
 
         if target_node != node:
             self.swap(node, target_node)
+            self.heapify(node)
```

Comment:
unable to catch this!

By definition the heapify method will be called to put the element in the right location, and in this case the node should be already in the right pos